cfgov/jinja2/v1/_includes/molecules/social-media.html

```diff
@@ -8,40 +8,47 @@
 
    Creates markup for Social Media molecule.
 
-   options:                  Object of optional parameters.
+   value:                   Object of optional parameters.
 
-   options.title:            Sets the title in email subject line
-                             and in the LinkedIn share post.
-                             Default is "Look what I found on the CFPB!"
+   value.blurb:             Sets the text of a tweet, the subject of an email,
+                            or the title of a LinkedIn post.
+                            Default is "Look what I found on the CFPB's site!"
 
-   options.is_share_view:    Whether the "Share this" heading is shown.
-                             Default is true.
+   value.is_share_view:     Whether the "Share this" heading is shown.
+                            Default is true.
 
-   options.twitter_hashtags: A comma-separated list of hashtags
-                             to be appended to default Tweet text.
-                             https://dev.twitter.com/web/tweet-button/parameters
+   value.twitter_hashtags:  A comma-separated list of hashtags
+                            to be appended to default Tweet text.
+                            https://dev.twitter.com/web/tweet-button/parameters
 
-   options.twitter_related:  A comma-separated list of accounts
-                             related to the content of the shared URI.
-                             https://dev.twitter.com/web/tweet-button/parameters
-                             Default is `cfpb`.
+   value.twitter_related:   A comma-separated list of accounts
+                            related to the content of the shared URI.
+                            https://dev.twitter.com/web/tweet-button/parameters
+                            Default is `cfpb`.
 
-   options.twitter_lang:     Loads text components in the specified language,
-                             if other than English.
-                             https://dev.twitter.com/web/tweet-button/parameters
+   value.twitter_lang:      Loads text components in the specified language,
+                            if other than English.
+                            https://dev.twitter.com/web/tweet-button/parameters
 
    ========================================================================== #}
 
+{# from 'macros/share.html' import share_twitter_url_new as _share_twitter_url #}
+
+{% macro render( value ) %}
+
 {# TODO: Remove global page reference when moving to Wagtail. #}
-{% macro render( options={} ) %}
-{% set title = options.title | urlencode or ( page or {} ).seo_title | urlencode  %}
+{% set blurb = value.blurb | urlencode or
```

Comment:
can you use parens and like ` value.blurb or page.seo_title or value.title or  'Look what I found on the CFPB’s site!' ) | urlencode?